cli/internal/graph/graph.go

```diff
@@ -152,6 +179,6 @@ func (g *CompleteGraph) GetPackageJSONFromWorkspace(workspaceName string) (*fs.P
 
 // repoRelativeLogFile returns the path to the log file for this task execution as a
 // relative path from the root of the monorepo.
-func repoRelativeLogFile(pt *nodes.PackageTask) string {
-	return filepath.Join(pt.Pkg.Dir.ToStringDuringMigration(), ".turbo", fmt.Sprintf("turbo-%v.log", pt.Task))
+func repoRelativeLogFile(dir turbopath.AnchoredSystemPath, taskName string) string {
```

Comment:
changed this so getting the logfile doesn't depend on the `packageTask` struct, since it's also used by taskSummary  now